src/librustc/ty/maps.rs

```diff
@@ -608,3 +609,11 @@ fn typeck_item_bodies_dep_node(_: CrateNum) -> DepNode<DefId> {
 fn const_eval_dep_node((def_id, _): (DefId, &Substs)) -> DepNode<DefId> {
     DepNode::ConstEval(def_id)
 }
+
+fn describe_def(def_id: DefId) -> DepNode<DefId> {
+    DepNode::DescribeDef(def_id)
+}
+
+fn def_span(def_id: DefId) -> DepNode<DefId> {
+    DepNode::DefSpan(def_id)
```

Comment:
These aren't needed, because they're just a `DepNode` variant - the explicit functions are in cases where you *can't* just use `DepNode::VariantName(def_id)`.